### PR TITLE
A few small modifications for readability

### PR DESCRIPTION
**Pull request by AndrewWant**
I just noticed a couple of areas where the code was a little hard to follow and left in a couple of suggestions as comments.

### CODE REVIEW

File: Apeer/fiji_scripts/fijipytools.py [hunk 1]
```diff
@@ -288,7 +277,8 @@ def readCZI(imagefile,
 
         # calc scaling in case of pyramid
         # scale = float(metainfo['Output SizeX']) / float(metainfo['SizeX'])
-        scale = float(metainfo['SizeX']) / float(metainfo['Output SizeX'])
+        scale = float(metainfo['SizeX']) / metainfo['Output SizeX']
+        # was this ported from python 2? I think you only need to float one of the inputs to the division
```
Comment (FelixS90) [inline Apeer/fiji_scripts/fijipytools.py]: You are right. Floating the first number is enough. If this code is intended to be only used in Py3, one could even leave out this conversion entirely. See https://stackoverflow.com/questions/1535596/what-is-the-reason-for-having-in-python

I would remove the comment though.

File: Apeer/fiji_scripts/fijipytools.py [hunk 2]
```diff
@@ -477,8 +468,10 @@ def apply_filter(imp, radius=5, filtertype='MEDIAN'):
             ip = stack.getProcessor(index)
 
             # apply filter based on filtertype
-            # if filtertype == 'MEDIAN':
-            filter.rank(ip, radius, filterdict[filtertype])
+            if filtertype in filterdict:
+                filter.rank(ip, radius, filterdict[filtertype])
+            else:
+                print(f"Argument 'filtertype': {filtertype} not found")
```
Comment (FelixS90) [inline Apeer/fiji_scripts/fijipytools.py]: That is a bit critical I would say. In the original version, you throw with a KeyError or something like that, which is pretty much the same as what you are printing in the `else` statement. However now, you silently ignore the error (except for this print statement, but this does not stop control flow and the caller might not even notice since it goes to stdout) and you might get subtle errors later in the code.

My suggestion: Either leave it as it is or do something meaningful, i.e. [only catch what you can/will handle](https://stackoverflow.com/questions/4673860/what-does-only-catch-exceptions-you-can-handle-really-mean).

File: Apeer/fiji_scripts/fijipytools.py [hunk 3]
```diff
@@ -633,27 +626,26 @@ class ThresholdTools:
...
-            lowthresh = Auto_Threshold.Mean(hist)
-        if method == 'Shanbhag':
-            lowthresh = Auto_Threshold.Shanbhag(hist)
-        if method == 'Yen':
-            lowthresh = Auto_Threshold.Yen(hist)
-        if method == 'Li':
-            lowthresh = Auto_Threshold.Li(hist)
-
-        return lowthresh
+        # I think the use of a dict here makes it a little easier to see what's going on, and avoids repetition
```
Comment (FelixS90) [inline Apeer/fiji_scripts/fijipytools.py]: Again, good catch. I would however remove the comment itself here.

File: Apeer/fiji_scripts/fijipytools.py [hunk 4]
```diff
@@ -195,33 +195,22 @@ def readbf(imagefile, metainfo,
...
-            metainfo['Output SizeY'] = height
-
-            imp = imps[0]
-
-        return imp
-
+        allowed_methods = {"IJ": IJ.openImage, 
+                           "Opener": Opener().openImage, 
+                           "BF": BF.openImagePlus}
+        if method in allowed_methods:
```
Comment (FelixS90) [inline Apeer/fiji_scripts/fijipytools.py]: Good catch.

I am just wondering whether it is better to just omit the `if method in allowed_methods:` entirely here. It essentially boils down to the question whether `None` should be returned in this case or whether an exception should be raised. If an exception should be raised that you do not catch anyways, you can also omit the if clause.

File: Apeer/fiji_scripts/fijipytools.py [hunk 5]
```diff
@@ -633,27 +626,26 @@ class ThresholdTools:
...
+                      }
+        if method in method_dict:
+            method_func = method_dict[method]
+            lowthresh = method_func(hist)
+            return lowthresh
+        else:
+            print(f"Method passed not found: {method}")
+            # or print("Method passed not found: {}".format(method) or print("Method passed not found: %s" % method)
+            # whichever works best for you
+            return None
```
Comment (FelixS90) [inline Apeer/fiji_scripts/fijipytools.py]: Same as above. Either throw or return None. In any case you can omit the if-else statement.

File: Apeer/fiji_scripts/fijipytools.py (hunk 4 above)
Comment (AndrewWant) [reply Apeer/fiji_scripts/fijipytools.py]: Committed with the omission as suggested

File: Apeer/fiji_scripts/fijipytools.py (hunk 5 above)
Comment (AndrewWant) [reply Apeer/fiji_scripts/fijipytools.py]: Same as above

File: Apeer/fiji_scripts/fijipytools.py (hunk 2 above)
Comment (AndrewWant) [reply Apeer/fiji_scripts/fijipytools.py]: Modified as suggested

File: Apeer/fiji_scripts/fijipytools.py (hunk 3 above)
Comment (AndrewWant) [reply Apeer/fiji_scripts/fijipytools.py]: I think I've modified everything you've asked me to take out... Apologies for the delay, this is my first pull request, so I'm kind of feeling my way around. Thanks for your patience :)
